app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :user
```

Comment:
- [ ] Kindly change the `belongs_to :user` to `belongs_to :author, class_name: 'User'` to match the associations between models according to the given [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd.png).